src/api/selection.js

```diff
@@ -35,91 +35,26 @@ function (elementHelper) {
       var range = this.range;
       if(!range) { return; }
 
-      var startMarker = document.createElement('em');
-      startMarker.classList.add('scribe-marker');
-      var endMarker = document.createElement('em');
-      endMarker.classList.add('scribe-marker');
-
-      // End marker
-      var rangeEnd = this.range.cloneRange();
-      rangeEnd.collapse(false);
-      rangeEnd.insertNode(endMarker);
-
-      /**
-       * Chrome and Firefox: `Range.insertNode` inserts a bogus text node after
-       * the inserted element. We just remove it. This in turn creates several
-       * bugs when perfoming commands on selections that contain an empty text
-       * node (`removeFormat`, `unlink`).
-       * As per: http://jsbin.com/hajim/5/edit?js,console,output
-       */
-      // TODO: abstract into polyfill for `Range.insertNode`
-      if (endMarker.nextSibling &&
-          endMarker.nextSibling.nodeType === Node.TEXT_NODE
-          && endMarker.nextSibling.data === '') {
-        endMarker.parentNode.removeChild(endMarker.nextSibling);
-      }
-
+      var scribeNodeRange = document.createRange();
+      scribeNodeRange.selectNodeContents(scribe.el);
 
+      //we want to ensure that the current selection is within the current scribe node
+      //if this isn't true scribe will place markers within the selections parent
+      //we want to ensure that scribe ONLY places markers within it's own element
+      var selectionStartWithinScribeElementStart = this.range.compareBoundaryPoints(Range.START_TO_START, scribeNodeRange) >= 0;
+      var selectionEndWithinScribeElementEnd = this.range.compareBoundaryPoints(Range.END_TO_END, scribeNodeRange) <= 0;
 
-      /**
-       * Chrome and Firefox: `Range.insertNode` inserts a bogus text node before
-       * the inserted element when the child element is at the start of a block
-       * element. We just remove it.
-       * FIXME: Document why we need to remove this
-       * As per: http://jsbin.com/sifez/1/edit?js,console,output
-       */
-      if (endMarker.previousSibling &&
-          endMarker.previousSibling.nodeType === Node.TEXT_NODE
-          && endMarker.previousSibling.data === '') {
-        endMarker.parentNode.removeChild(endMarker.previousSibling);
-      }
+      if (selectionStartWithinScribeElementStart && selectionEndWithinScribeElementEnd){
 
+        var startMarker = document.createElement('em');
+        startMarker.classList.add('scribe-marker');
+        var endMarker = document.createElement('em');
+        endMarker.classList.add('scribe-marker');
```

Comment:
Maybe you were thinking of the helpers in the noting plugin?  But yeah, would be a nice addition.